src/internal/operations/test/sharepoint_test.go

```diff
@@ -150,6 +150,13 @@ func (suite *SharePointBackupIntgSuite) TestBackup_Run_sharePointBasic_groups9Ve
 		"items written")
 }
 
+func (suite *SharePointBackupIntgSuite) TestBackup_Run_sharePointBasic_assistBackup_groups9VersionBump() {
```

Comment:
Question: why is sharepoint getting a special test case here?  It shouldn't be affected by the bump.

Edit: now that I read the full test func, it seems like this is the negative case.  Do we want a negative case test for all services?

Reply:
This attempts to add a negative test case for as many services as possible. With the current implementation, it turns into "as many services as support extensions" because we need to inject a recoverable error in the backup. Since OneDrive is misbehaving for other reasons that negative test case is commented out leaving us with tests running for SharePoint and groups/teams